tests/select/test_drop_pv_generating_overnight.py

```diff
@@ -0,0 +1,80 @@
+from timeit import timeit
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from ocf_datapipes.select import DropNightPV
+from ocf_datapipes.transform.xarray import AssignDayNightStatus
+
+
+def test_drop_with_passiv_datapipe(passiv_datapipe):
+    before_dropping_pv_with_night_output = AssignDayNightStatus(passiv_datapipe)
+    after_dropping_pv_with_night_output = DropNightPV(before_dropping_pv_with_night_output)
+
+    data_before_drop = next(iter(before_dropping_pv_with_night_output))
+    data_after_drop = next(iter(after_dropping_pv_with_night_output))
+    after_id = data_after_drop.coords["pv_system_id"].values
+
+    Num_of_sys_before_drop = len(data_before_drop.coords["pv_system_id"].values)
+    Num_of_sys_after_drop = len(data_after_drop.coords["pv_system_id"].values)
+
+    # In 'pvoutput_datapipe', there are 41 or so systems and some of the systems have been dropped
+    # as they generate power over night.
+    print("\nTest1")
```

Comment:
Possibly convert to logging statements, so that the test outputs are not as messy, for this and the other tests

Reply:
yes, that makes sense